wal/src/rocks_impl/manager.rs

```diff
@@ -566,11 +623,42 @@ impl Builder {
             None
         };
 
-        let db = DB::open(rocksdb_config, &self.wal_path)
-            .map_err(|e| e.into())
-            .context(Open {
-                wal_path: self.wal_path.clone(),
-            })?;
+        let mut cf_opts = ColumnFamilyOptions::new();
+        if let Some(v) = self.write_buffer_size {
+            cf_opts.set_write_buffer_size(v);
+        }
+        if let Some(v) = self.max_write_buffer_number {
+            cf_opts.set_max_write_buffer_number(v);
+        }
+        if let Some(v) = self.level_zero_file_num_compaction_trigger {
+            cf_opts.set_level_zero_file_num_compaction_trigger(v);
+        }
+        if let Some(v) = self.level_zero_slowdown_writes_trigger {
+            cf_opts.set_level_zero_slowdown_writes_trigger(v);
+        }
+        if let Some(v) = self.level_zero_stop_writes_trigger {
+            cf_opts.set_level_zero_stop_writes_trigger(v);
+        }
+
+        // FIFO compaction strategy let rocksdb looks like a message queue.
+        if let Some(v) = self.fifo_compaction_max_table_files_size {
+            if v > 0 {
+                let mut fifo_opts = FifoCompactionOptions::new();
+                fifo_opts.set_max_table_files_size(v);
+                cf_opts.set_fifo_compaction_options(fifo_opts);
+                cf_opts.set_compaction_style(DBCompactionStyle::Fifo);
+            }
+        }
+
+        let db = DB::open_cf(
+            rocksdb_config,
+            &self.wal_path,
+            vec![(ROCKSDB_DEFAULT_COLUMN_FAMILY, cf_opts)],
+        )
```

Comment:
Maybe we can use `ColumnFamilyDescriptor::default()` to create the descriptor of default column family:
```suggestion
        let default_cfd = {
            let mut cfd = ColumnFamilyDescriptor::default();
            cfd.options = cf_opts;
            cfd
         };
        let db = DB::open_cf(
            rocksdb_config,
            &self.wal_path,
            vec![default_cfd],
        )
```